packages/components/socialFeed/RichText/Toolbar/ToolbarContainer.tsx

```diff
@@ -6,26 +6,33 @@ import {
   gradientColorTurquoise,
 } from "../../../../utils/style/colors";
 import { layout } from "../../../../utils/style/layout";
-import { LegacyPrimaryBox } from "../../../boxes/LegacyPrimaryBox";
+import { Box } from "../../../boxes/Box";
 
 export const ToolbarContainer: FC<{ children: ReactNode }> = ({ children }) => {
   return (
-    <LegacyPrimaryBox
-      fullWidth
-      colors={[
-        gradientColorDarkerBlue,
-        gradientColorBlue,
-        gradientColorTurquoise,
-      ]}
-      style={{ flex: 1 }}
-      mainContainerStyle={{
+    <Box
+      borderGradient={{
+        start: { x: 0, y: 0 },
+        end: { x: 1, y: 1 },
+        locations: [0.7, 0.8],
+        colors: [
+          gradientColorDarkerBlue,
+          gradientColorBlue,
+          gradientColorTurquoise,
+        ],
+      }}
```

Comment:
there's a regression here
before:
<img width="1002" alt="Screenshot 2023-12-21 at 19 24 11" src="https://github.com/TERITORI/teritori-dapp/assets/7917064/9dbba73e-7439-454d-abc0-80a896983fbd">
after:
<img width="1005" alt="Screenshot 2023-12-21 at 19 24 36" src="https://github.com/TERITORI/teritori-dapp/assets/7917064/68f2ce9b-61b2-48a0-92f7-0805ff080d2a">

Reply:
@n0izn0iz Thanks for notifying, The issue is fixed now.